OxfordBodleian/Juel-Jensen/BDLaethf24.xml

```diff
@@ -152,25 +151,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </decoNote>
                             </binding>
                         </bindingDesc>
-                        
+
                     </physDesc>
-                    
+
                     <history>
                         <origin>
                             <origDate notBefore="1800" notAfter="1949" evidence="lettering"/>
-                            The manuscript probably consists of three distinct units of production: <ref target="#p1"/> and <ref target="#p2"/>, 
+                            The manuscript probably consists of three distinct units of production: <ref target="#p1"/> and <ref target="#p2"/>,
                             and <ref target="#p3"/>. <ref target="#p3"/>
                             consists of the addition of content on the material support of <ref target="#p1"/> and <ref target="#p2"/>
-                            and of the addition of one more quire, <ref target="#q6"/> (corresponding to the 
-                            transformation model A3 according to <bibl><ptr target="bm:Andrist2013Syntaxe"/><citedRange unit="page">65</citedRange></bibl>). 
+                            and of the addition of one more quire, <ref target="#q6"/> (corresponding to the
```

Comment:
Will the element `<ref target="#q6"/>` in this position generate any problems in the PDF output? @thea-m

Reply:
I think so, yes, as will all other empty refs...it is on the list of PDF problems